projects/demo/src/modules/app/pages.ts

```diff
@@ -941,7 +941,7 @@ export const pages: DocRoutePages = [
             },
             {
                 section: 'Components',
-                title: 'InputSlider [deprecated]',
+                title: 'InputSlider ⛔',
                 keywords: 'поле, инпут, форма, ввод, input, range, slider, диапазон',
                 route: DemoRoute.InputSliderLegacy,
```

Comment:
```suggestion
                title: 'InputSlider',
                keywords: 'поле, инпут, форма, ввод, input, range, slider, диапазон',
                route: DemoRoute.InputSlider,
```

It is inside this group:
https://github.com/taiga-family/taiga-ui/blob/be3e4c422f4bdd59faab99e99964f8cf6947ee0d/projects/demo/src/modules/app/pages.ts#L932-L962